tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -114,6 +117,15 @@ def test_params_default(self):
                     self.Parameter(name='test', desc='  test description2 \n')]
         self.check_docstring(doc, expected)
 
+    def test_excps_default(self):
```

Comment:
write exception properly instead of `excps`.

And why have a 'default' with doc like ` :raises test:  test description1 \n` , which has inappropriate whitespace everywhere.
Why not `    :raises test: test description1\n` which is proper whitespace.

Then add other tests with weird extra whitespace to show how the case behaves when it sees invalid whitespace.